manifests/services/dashboard/apache.pp

```diff
@@ -1,12 +1,20 @@
 # Configures the apache vhost for the dashboard.
 class profile::services::dashboard::apache {
-  $configfile = hiera('profile::dashboard::configfile',
-      '/etc/shiftleader/settings.ini')
-  $dashboardname = hiera('profile::dashboard::name')
-  $dashboardv4name = hiera('profile::dashboard::name::v4only', false)
-  $management_if = hiera('profile::interfaces::management')
+  $configfile = lookup('profile::dashboard::configfile', {
+    'value_type'    => Stdlib::Absolutepath,
+    'default_value' => '/etc/shiftleader/settings.ini'
+    })
```

Comment:
Indentering: }) bør kanskje flyttes to hakk til venstre? 3 steder i denne fila og.